velox/common/memory/HashStringAllocator.cpp

```diff
@@ -99,6 +101,41 @@ void HashStringAllocator::clear() {
   for (auto i = 0; i < kNumFreeLists; ++i) {
     new (&free_[i]) CompactDoubleList();
   }
+
+  static const auto kHugePageSize = memory::AllocationTraits::kHugePageSize;
```

Comment:
Is this a bit expensive to do so? Maybe we could only check this in a non-debug build?

Reply:
I have added `#ifdef NDEBUG` check.